docs/notes/migration_guide.rst

```diff
@@ -209,6 +210,8 @@ If your project previously included any of these headers, you will need to modif
 Since these headers are now private, you should replace their usage with public alternatives or refactor the
 related code to ensure it does not depend on private headers.
 
+.. TODO:: Add a note about which headers to use instead of the private ones.
+  
```

Comment:
I believe this is way too brief. Shouldn't we explain what they should include instead?